velodyne_driver/src/driver/driver.cc

```diff
@@ -107,11 +111,45 @@ bool VelodyneDriver::poll(void)
 {
   // Allocate a new shared pointer for zero-copy sharing with other nodelets.
   velodyne_msgs::VelodyneScanPtr scan(new velodyne_msgs::VelodyneScan);
-  scan->packets.resize(config_.npackets);
 
+  if( config_.cut_angle >= 0) //Cut at specific angle feature enabled
+  {
+    scan->packets.reserve(config_.npackets);
+    velodyne_msgs::VelodynePacket tmp_packet;
+    while(true)
+    {
+      while(true)
+      {
+        int rc = input_->getPacket(&tmp_packet);
```

Comment:
The API has changed here. getPacket now expects two arguments.
https://github.com/ros-drivers/velodyne/blob/3747e937cfc1b2986c0b2e4983bdd0d7e8934079/velodyne_driver/include/velodyne_driver/input.h#L80-L81

Reply:
This only leads to an error, if you try to merge this feature into the latest master branch. Please see https://github.com/ros-drivers/velodyne/pull/126 for an updated version of this feature.